### PR TITLE
CASMINST-5258 Linting/formatting

### PR DESCRIPTION
**Pull request by rustydb**
# Description

Various linting with some opinionated formatting for the new/alpha "Add NCN Data page."

# Checklist Before Merging

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md [hunk 1]
```diff
@@ -422,38 +489,38 @@ Scenarios where this procedure is applicable:
...
+    > 1. (`ncn-mw#`) Verify that the BMC is reachable at the expected IP address.
+    >
+    >    ```bash
+    >    ping "${NODE}-mgmt"
+    >    ```
+    >
+    > 1. (`ncn-mw#`) Verify that the BMC Redfish `v1/Managers` endpoint is reachable.
+    >
+    >    ```curl
+    >    curl -f -u root:$IPMI_PASSWORD https://${BMC_XNAME}/redfish/v1/Managers
```
Comment (rsjostrand-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md:522]: Like my comment on the other PR, this should be `-k`

**Review by rsjostrand-hpe — CHANGES_REQUESTED** [review]

**Review by rsjostrand-hpe — APPROVED** [review]

File: operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # Add NCN Data
 
+## Topics
+
+* [Description](#description)
+* [Prerequisites](#prerequisites)
+* [Procedure](#procedure)
+  * [Collect information from the NCN](#collect-information-from-the-ncn)
+    * [Saving/Reloading](#savingreloading)
+  * [Collect MAC Addresses from the NCN](#collect-mac-addresses-from-the-ncn)
```
Comment (mharding-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md:10]:
```suggestion
  * [Collect MAC addresses from the NCN](#collect-mac-addresses-from-the-ncn)
```

File: operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+    * [Swapping/Moving an NCN](#swappingmoving-an-ncn)
```
Comment (mharding-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md:11]:
```suggestion
    * [Swapping/moving an NCN](#swappingmoving-an-ncn)
```

**Review by mharding-hpe — APPROVED** [review]

File: operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md [hunk 4]
```diff
@@ -1,5 +1,17 @@
 # Add NCN Data
 
+## Topics
+
+* [Description](#description)
+* [Prerequisites](#prerequisites)
+* [Procedure](#procedure)
+  * [Collect information from the NCN](#collect-information-from-the-ncn)
+    * [Saving/Reloading](#savingreloading)
```
Comment (mharding-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md:9]:
```suggestion
    * [Saving/reloading](#savingreloading)
```

**Review by mharding-hpe — APPROVED** [review]

**Pull request merged**